packages/fiori/src/MediaGalleryItem.ts

```diff
@@ -175,9 +46,123 @@ const metadata = {
  * @implements sap.ui.webc.fiori.IMediaGalleryItem
  * @since 1.1.0
  */
-class MediaGalleryItem extends UI5Element {
+@customElement("ui5-media-gallery-item")
+class MediaGalleryItem extends UI5Element implements ITabbable {
+	/**
+	 * Defines the selected state of the component.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.MediaGalleryItem.prototype.selected
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	selected!: boolean;
+
+	/**
+	 * Defines whether the component is in disabled state.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.MediaGalleryItem.prototype.disabled
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Determines the layout of the item container.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>Square</code></li>
+	 * <li><code>Wide</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.fiori.types.MediaGalleryItemLayout}
+	 * @name sap.ui.webc.main.MediaGalleryItem.prototype.layout
+	 * @defaultvalue "Square"
+	 * @public
+	 */
+	@property({ type: MediaGalleryItemLayout, defaultValue: MediaGalleryItemLayout.Square })
+	layout!: MediaGalleryItemLayout;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	_interactive!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	_square!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	_contentImageNotFound!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	_thumbnailNotFound!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	_thumbnailDesign!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ type: Boolean })
+	focused!: boolean;
+
+	/**
+	 * @private
+	 */
+	@property({ defaultValue: "" })
```

Comment:
@property decorator adds "" as default value by default. We cant remove the object passed to the decorator.